shell/platform/android/test/io/flutter/FlutterTestSuite.java

```diff
@@ -29,6 +29,7 @@
 import test.io.flutter.embedding.engine.FlutterShellArgsTest;
 import test.io.flutter.embedding.engine.PluginComponentTest;
 import test.io.flutter.embedding.engine.dart.DartExecutorTest;
+import io.flutter.view.AccessibilityBridgeTest;
```

Comment:
I assume the formatter will complain about this

Reply:
perhaps ... I will format another day :)